content/en/blog/_posts/2023-01-06-unhealthy-pod-eviction-policy-for-pdb.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: blog
+title: "Eviction policy for unhealthy pods guarded by PodDisruptionBudgets"
+date: 2023-01-06
+slug: "unhealthy-pod-eviction-policy-for-pdbs"
+---
+
+**Authors:** Filip Křepinský (Red Hat), Morten Torkildsen (Google), Ravi Gudimetla (Apple)
+
+## What problems does this solve?
+
+Eviction of pods respects Pod Disruption Budgets (PDBs). This means that the eviction of a pod
```

Comment:
Link to https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets

Reply:
If making the change suggested in https://github.com/kubernetes/website/pull/37627/files#r1053221368, you could put the link there instead.

Reply:
added the link to the intro